dkg/sync/client.go

```diff
@@ -85,10 +85,10 @@ func (c *Client) Run(ctx context.Context) error {
 		relayBroke, connBroke, err := c.sendMsgs(ctx, stream)
 		c.clearConnected()
 		if relayBroke {
-			log.Debug(ctx, "Relay connection dropped, reconnecting")
+			log.Warn(ctx, "Relay connection dropped, reconnecting", nil, z.Str("peer", p2p.PeerName(stream.Conn().RemotePeer())))
 			continue // Always reconnect on relay circuit recycling.
 		} else if connBroke && c.reconnect {
-			log.Debug(ctx, "Connection dropped, reconnecting")
+			log.Warn(ctx, "Connection dropped, reconnecting", nil, z.Str("peer", p2p.PeerName(stream.Conn().RemotePeer())))
```

Comment:
peer name also added to ctx above. Not sure the stack traces of WARN level is useful, maybe just info level? Suggest: `log.Info(ctx, "Disconnected from peer")`

Reply:
peer name added in ctx in local peer and not remote peer. Here's the screenshot with current WARN logs:
<img width="1302" alt="Screenshot 2022-12-14 at 12 19 10 PM" src="https://user-images.githubusercontent.com/37813203/207526113-a24059a8-7e3a-474f-a2df-3fbe48fde140.png">

Reply:
New logs:
![Uploading Screenshot 2022-12-14 at 12.35.32 PM.png…]()

Reply:
when is the `WARN: Error serving sync protocol` logged?
when is the `ERRO: Fatal error...` logged?

Reply:
The aim is to make it look nice:
 - Peer disconnecting isn't such a big problem, a simple "peer disconnected" info log is fine.
 - Relay connections are recycled every 2min, that shouldn't result in any logs (only simple debug) 
 - Warnings and Errors should only happen when things are crashing